### PR TITLE
Fix asset loading when running develop inside github codespaces

### PR DESCRIPTION
**Pull request by maman**
## Summary

The default webpack devserver asset loading mechanism breaks when running in development mode inside github codespaces, because it'll try to load assets from `localhost`.

This PR fixes that by overriding the default devServer's `host` and `publicPath` to match the current codespaces workspace.

### CODE REVIEW

File: razzle.config.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const IS_CODESPACES = process.env.CODESPACES === "true";
+const CODESPACES_NAME = process.env.CODESPACE_NAME;
+
+module.exports = {
+    modifyWebpackConfig(opts) {
+        const webpackConfig = opts.webpackConfig;
+        if (opts.env.target === 'web' && opts.env.dev && IS_CODESPACES) {
+            const webpackConfig = opts.webpackConfig;
+            const host = `${CODESPACES_NAME}-3001.githubpreview.dev`;
```
Comment (ans-4175) [inline razzle.config.js:9]: ini env-nya dapat darimana @maman ?

File: razzle.config.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            webpackConfig.devServer.host = host;
+            webpackConfig.output.publicPath = `https://${host}/`;
```
Comment (ans-4175) [inline razzle.config.js:11]: oke dipaksa ke host begini

File: razzle.config.js (hunk 1 above)
Comment (maman) [reply razzle.config.js:9]: itu otomatis kedefine didalam codespaces env nya sih. coba cek aja:
```
$ env | grep CODESPACE
```

Comment (ans-4175) [reply razzle.config.js:9]: ohya, nice pak.

**Review by ans-4175 — APPROVED** [review]
LGTM

**Pull request merged**